WellLog.Lib/Helpers/ByteHelpers.cs

```diff
@@ -0,0 +1,23 @@
+﻿using System;
+
+namespace WellLog.Lib.Helpers
+{
+    public static class ByteHelpers
+    {
+        public static bool GetBit(this byte b, byte mask)
+        {
+            return (b & mask) > 0;
+        }
+
+        public static byte SetBit(this byte b, byte mask, bool value)
+        {
+            if (value) { return Convert.ToByte(b | mask); }
```

Comment:
Remove the braces or add the lines. Using braces on one line is just clutter.

Reply:
[Reflections on Curly Braces – Apple’s SSL Bug and What We Should Learn From It](https://blog.codecentric.de/en/2014/02/curly-braces/)

fuck that.  i love curly braces.

Reply:
Either way, as long as you're consistent.  Since you're T-Frels, I know it is without looking.